### DISCUSSION TITLE
Books names translation

### DISCUSSION THREAD

**Discussion by debritto** [ezra-bible-app/ezra-bible-app#538] [Q&A] [answered]
Hi,
Is it possible to translate the Books names?

![image](https://user-images.githubusercontent.com/9845780/142950885-cc4ca27f-fb80-4816-8ae2-2a9e7e4b7442.png)

Best regards

**Comment by tobias-klein** [comment, marked as answer]
Yes, absolutely - and there are translations available for quite some languages. These translations however are coming from the SWORD library. In case of Ezra Bible App we are using node-sword-interface to access the SWORD library and that also contains a copy of those translations:
https://github.com/ezra-bible-app/node-sword-interface/tree/master/locales.d

As I am looking at it - it seems like Brazilian Portuguese is  available, but not recognized in Ezra Bible App. That's a bug. I'll create a separate issue for this.
